Invoro.WebApp/Invoro/invoro/src/lib/components/Status/Status.component.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { mount, shallow, ReactWrapper } from 'enzyme';
+import StatusComponent from './Status.component';
+import Status from '../../dataModel/Status';
+
+describe("StatusComponent", () => {
+    it("when status is NotPlanned, the label is corresponding", () => {
+        // Prepare
+        let props = {
+            status: Status.NotPlanned
+        };
+
+        // Action
+        let statusComponent: ReactWrapper = mount(<StatusComponent {...props} />);
```

Comment:
Is 'mount' nessesary? maybe shallow is enough because you render here only the parnet component

Reply:
shallow doesnt work, since VoteComponent renders div which contains the chip, and shallow doesnt render the chip, so I can't test if the chip is correct